econml/inference.py

```diff
@@ -118,17 +118,21 @@ def const_marginal_effect_inference(self, X):
             X = self.featurizer.transform(X)
         X, T = broadcast_unit_treatments(X, self.d_t)
         pred = reshape_treatmentwise_effects(self._predict(cross_product(X, T)), self._d_t, self._d_y)
-        if not hasattr(self.model_final, 'prediction_stderr'):
-            raise AttributeError("Final model doesn't support prediction standard eror, "
-                                 "please call const_marginal_effect_interval to get confidence interval.")
-        pred_stderr = reshape_treatmentwise_effects(self._prediction_stderr(cross_product(X, T)), self._d_t, self._d_y)
+        pred_stderr = None
+        if hasattr(self.model_final, 'prediction_stderr'):
+            pred_stderr = reshape_treatmentwise_effects(self._prediction_stderr(cross_product(X, T)),
+                                                        self._d_t, self._d_y)
```

Comment:
Since _prediction_stderr already has the check, maybe

```suggestion
        pred_stderr = self._prediction_stderr(cross_product(X, T))
        if pred_stderr is not None:
            pred_stderr = reshape_treatmentwise_effects(pred_stderr,
                                                        self._d_t, self._d_y)
```

unless we don't want to see the warning in this case.

Reply:
I just added the warning message there too. I didn't want to perform the crossproduct only to then return None.